client/app/controllers/welcome.js

```diff
@@ -0,0 +1,40 @@
+import Ember from 'ember';
+
+export default
+Ember.Controller.extend({
+
+    websocket: Ember.inject.service(),
+
+    subscribe: function() {
+        var controller = this;
+        let socket = controller.get('websocket.socket');
+
+        var self = this;
+        // Add a connect listener
+        socket.on('connect info', function (data) {
+            console.log('Connect Info Received ID: ', data);
+            self.get("websocket").playerID = data;
+        });
+
+        // Add a connect listener
+        socket.on('user connect', function (data) {
+            console.log('Received a user connect from the server:\n', data);
+            self.transitionToRoute("game");
+        });
+
+    },
+
+    unsubscribe: function() {
+        let socket = this.get('websocket.socket');
+        socket.removeAllListeners();
+    },
+
+    actions: {
+        "Start Game": function () {
```

Comment:
Interesting, don't usually see actions with spaces in the name. But i suppose it doesn't matter anymore now that it is required in your templates to have quotes around the action name